tests/features/custom_page/collection.custom_page.feature

```diff
@@ -0,0 +1,44 @@
+@api
+Feature: "Add custom page" visibility options.
+  In order to manage custom pages
+  As a collection member
+  I need to be able to add "Custom Page" content through UI.
+  Background:
+    Given users:
+    | name            | mail                  | roles         | pass |
+    | Rufus Drumknott | member@example.com    |               | test |
+    And the following collections:
+    | uri                             | name           | description               | logo     |  owner |
+    | http://joinup.eu/collection/foo | Foo Collection | This is a foo collection. | logo.png |        |
+    | http://joinup.eu/collection/bar | Bar Collection | This is a bar connection. | logo.png |        |
+    And the following user memberships:
+    | group_type | group_id                         | member          |
+    | rdf_entity | http://joinup.eu/collection/foo  | Rufus Drumknott |
+
+  Scenario: Check visibility of the "Add custom page" button.
+    When I am logged in as "Rufus Drumknott"
+    And I go to the homepage of the "Foo Collection" collection
+    Then I should see the link "Add custom page"
+    When I click "Leave this collection"
+    And I press the "Confirm" button
+    Then I should not see the link "Add custom page"
+    When I go to the homepage of the "Bar Collection" collection
+    Then I should not see the link "Add custom page"
+    But I should see the button "Join this collection"
+    When I press the "Join this collection" button
+    Then I should see the link "Add custom page"
+
+  Scenario: Add custom page as a collection member.
+    When I am logged in as "Rufus Drumknott"
+    And I go to the homepage of the "Foo Collection" collection
+    Then I should see the link "Add custom page"
+    When I click "Add custom page"
+    Then I should see the text "Add custom page"
```

Comment:
This test is a bit meaningless. This will simply check if the text "Add custom page" is visible anywhere on the page. This might be a link, a menu item, even a string somewhere in the body text. I suppose you are checking here to see that the page title is "Add custom page" - for that case you can use "Then I should see the heading :heading".